README.md

```diff
@@ -1,139 +1,32 @@
 # Kritis
 
-Kritis (“judge” in Greek), provides full software supply chain security for Kubernetes applications,
-allowing devOps teams to enforce deploy-time image security policies using metadata and attestations stored in [Grafeas](https://github.com/grafeas/grafeas).
+Kritis (“judge” in Greek), is an open-source solution for software supply chain security for Kubernetes applications. Kritis enforces deploy-time security policies using the [Google Cloud Container Analysis API](https://cloud.google.com/container-analysis/api/reference/rest/), and in a subsequent release, [Grafeas](https://github.com/grafeas/grafeas).
```

Comment:
maybe "is an open-source solution for securing your software supply chain for Kubernetes applications"

Reply:
Done.